app/src/main/res/xml/pref_misc.xml

```diff
@@ -67,10 +67,12 @@
             android:defaultValue="false"
             android:key="key_misc_disable_buttons_backlight"
             android:title="Disable backlight of hardware buttons if present (requires reboot)" />
-		<CheckBoxPreference
-            android:defaultValue="false"
-            android:key="key_misc_backlight_scale"
-            android:title="Force alternative backlight scale" />
+		<ListPreference
+			android:defaultValue="-1"
```

Comment:
This works fine? I'd expect it to require "system default"? Can you wipe data of treble app, and check the behaviour?

Reply:
i added  the logcat debug... every value works as expected. you need only a reboot if changing to system default (off -> system default in my case) because the rw system value only get applied on boot.